Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu/mysql:8.0-20.04_beta
+
+ENV CONFIG_VERSION="0.8.22-1"
+
+RUN set -eux && \
+	apt-get update && \
+	apt-get install -y wget lsb-release gnupg
+
+RUN set -eux && \
+	wget -q https://dev.mysql.com/get/mysql-apt-config_${CONFIG_VERSION}_all.deb \
+        -O /tmp/mysql-apt-config_${CONFIG_VERSION}_all.deb && \
+    DEBIAN_FRONTEND=noninteractive dpkg -i /tmp/mysql-apt-config_${CONFIG_VERSION}_all.deb && \
+    DEBIAN_FRONTEND=noninteractive apt-get update && \
```

Comment:
You've already run `apt-get update` in the last layer

Reply:
the first `apt-get update` is to install `wget` to be able to use it to pull the deb file. the mysql docs recommended running `apt-get update` again as we are running `dpkg -i` in the previous line. without the `apt-get update` again, we will be unable to install `mysql-shell` in the next line

Reply:
You could probably use curl, which I think is included by default, too :)

Reply:
On ubuntu image curl it's not included, and curl pull more dependencies than wget. Don't know about the "ubuntu/mysql" though.